docs/csv-description.md

```diff
@@ -220,20 +220,20 @@ If the `koku-metrics-operator` is configured to run in a restricted network, the
   $ oc delete -f volume-shell.yaml
   ```
 
-## Create a Source
-In a restricted network, the `koku-metrics-operator` cannot automatically create a source. This process must be done manually. In the console.redhat.com platform, open the [Sources menu](https://console.redhat.com/settings/sources/) to begin adding an OpenShift source to Cost Management:
+## Create an Integration
+In a restricted network, the `koku-metrics-operator` cannot automatically create an integration. This process must be done manually. In the console.redhat.com platform, open the [Integrations menu](https://console.redhat.com/settings/integrations/) to begin adding an OpenShift integration to Cost Management:
 
 Prerequisites:
 * The cluster identifier which can be found in the KokuMetricsConfig CR, the cluster Overview page, or the cluster Help > About.
 
-Creating a source:
+Creating an integration:
 1. Navigate to the Sources menu
```

Comment:
I believe this menu will change to integrations on Nov 1. So it should be Navigate to Integrations menu.

Reply:
Update! Thanks for the 👀